pom.xml

```diff
@@ -334,6 +334,11 @@
         <artifactId>json</artifactId>
         <version>20211205</version>
       </dependency>
+      <dependency>
+        <groupId>org.jsoup</groupId>
```

Comment:
Since this is in `dependencyManagement`, shouldn't it actually go to `io.jenkins.tools.bom` to resolve it upstream?

Reply:
the `io.jenkins.tools.bom` only contains plugin dependencies.  jsoup is a regular jar that is being bundled by this plugin.

Reply:
But I don't get it, this PR _add_ that dependency, not bumps the version of it.
Why do we need this dependency if it wasn't needed before?

From the PR title I'm guessing this is fixing some security issue (that I don't see any report linked to), but then I guess this dependency indirectly coming from another dependency?

Reply:
good question

firstly this is not declaring a `dependency` as it is inside the `dependencyManagent` section so it is just saying "regardless of what version any transitive dependency wants of this, it will get 1.14.3".  [ref](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management)

secondly - why is this even here?

looking at the dependency tree for the `master` branch I only see the plugin as in `test` scope.

checking out the `jira-3.6` and building I do not see jsoup being embedded in the HPI either

```
[INFO] Copy webapp webResources to C:\workarea\source\github\jenkinsci\jira-plugin\target\jira
[INFO] Assembling webapp jira in C:\workarea\source\github\jenkinsci\jira-plugin\target\jira
[WARNING] Bundling transitive dependency atlassian-event-3.0.0.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency atlassian-httpclient-api-2.0.0.jar (via jira-rest-java-client-api)
[INFO] Bundling direct dependency jira-rest-java-client-api-5.2.2.jar
[INFO] Bundling direct dependency jira-rest-java-client-core-5.2.2.jar
[WARNING] Bundling transitive dependency sal-api-3.0.7.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency jsr305-3.0.0.jar (via jira-rest-java-client-api)
[WARNING] Bundling transitive dependency error_prone_annotations-2.5.1.jar (via guava)
[WARNING] Bundling transitive dependency failureaccess-1.0.1.jar (via guava)
[INFO] Bundling direct dependency guava-30.1.1-jre.jar
[WARNING] Bundling transitive dependency listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar (via guava)
[WARNING] Bundling transitive dependency j2objc-annotations-1.3.jar (via guava)
[WARNING] Bundling transitive dependency jersey-client-1.19.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency jersey-core-1.19.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency jersey-json-1.19.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency jaxb-impl-2.2.3-1.jar (via jira-rest-java-client-core)
[INFO] Bundling direct dependency fugue-3.0.0.jar
[INFO] Bundling direct dependency atlassian-util-concurrent-4.0.1.jar
[WARNING] Bundling transitive dependency jsr311-api-1.1.1.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency jaxb-api-2.2.2.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency joda-time-2.9.9.jar (via jira-rest-java-client-api)
[INFO] Bundling direct dependency commons-lang3-3.12.0.jar
[INFO] Bundling direct dependency maven-artifact-3.8.2.jar
[WARNING] Bundling transitive dependency checker-qual-3.8.0.jar (via guava)
[WARNING] Bundling transitive dependency jettison-1.1.jar (via jira-rest-java-client-core)
[WARNING] Bundling transitive dependency plexus-utils-3.2.1.jar (via maven-artifact)
[WARNING] Bundling transitive dependency slf4j-api-1.7.30.jar (via jira-rest-java-client-core)
[INFO] Generating hpi C:\workarea\source\github\jenkinsci\jira-plugin\target\jira.hpi
```

if I go back to `jira-3.5` which could have been the latest at the time I also do not see it being bundled.


@olamy?